src/dotnet/commands/dotnet-add/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+using NuGet.Frameworks;
+using Microsoft.DotNet.Tools.Add.ProjectToProjectReference;
+
+namespace Microsoft.DotNet.Tools.Add
+{
+    public class AddCommand
+    {
+        public const string HelpText = @".NET Add Command
+
+Usage: dotnet add [options] [object] <command> [[--] <arg>...]]
```

Comment:
I think this is somewhat confusing. Should the help come in the same order as the parameters are provided above?

Like, we show options first, should it be the first in the help below?

Also, we put both object and command as arguments with <> but in the string above, object has [] while command has <>.